### PR TITLE
Remove SlowQuery struct in lieu of a map

### PR DESCRIPTION
**Pull request by christineyen**
We're building a map[string]interface{} for the `event.Event` ultimately anyway,
so we may as well use it directly in the first place. This ensures that
we omit missing values, rather than sending along zero-values in the
struct itself.

### CODE REVIEW

File: parsers/mysql/mysql.go [hunk 1]
```diff
@@ -325,38 +331,38 @@ func (p *Parser) handleEvent(rawE []string) (SlowQuery, time.Time) {
 				"line":  line,
 				"event": rawE,
 			}).Debug("readmin ping detected; skipping this event")
-			sq.skipQuery = true
+			return nil, time.Time{}
```
Comment (toshok) [inline parsers/mysql/mysql.go]: does `len(nil)` == 0 (line 288)?  because if so, cool.

Comment (christineyen) [inline parsers/mysql/mysql.go]: it does when it's a nil masquerading as a `map[string]interface{}`! one of the hilariously fun failures of `reflect.DeepEqual` vs `==`.

https://play.golang.org/p/JHTuJsDgLn

Comment (toshok) [inline parsers/mysql/mysql.go]: wow, love it

**Comment by toshok** [comment]
🎉

**Pull request merged**